RIAssigner/data/MatchMSData.py

```diff
@@ -116,6 +116,38 @@ def retention_indices(self, values: Iterable[Data.RetentionIndexType]):
         else:
             raise ValueError('There is different numbers of computed indices and peaks.')
 
+    @property
+    def comment(self) -> Iterable[Data.CommentFieldType]:
+        """ Get comments."""
+        self.comment_keys = "comment"
+        content = [safe_read_comment_key(spectrum, self.comment_keys) for spectrum in self._spectra]
+        return content
+    
+def safe_read_comment_key(spectrum: Spectrum, key: str) -> Optional[str]:
+    """ Read key from spectrum and convert to str or return 'None'.
+    Read the given key from the spectrum metadata and convert it to a string.
+    In case an exception is thrown or the key is not present, returns 'None'.
+
+    Parameters
+    ----------
+    spectrum:
+        Spectrum from which to read the key.
+    key:
+        Key to be read from the spectrum metadata.
+
+    Returns
+    -------
+        The key's value converted to string or 'None'.
+    """
+
+    meta_value = spectrum.get(key, default=None)
+    if meta_value is not None:
+        try:
+            meta_value = str(meta_value)
+        except ValueError:
+            meta_value = None
+    return meta_value
```

Comment:
This is not needed as we expect a string to be present here

Reply:
Addressed [b5244e3](https://github.com/RECETOX/RIAssigner/pull/112/commits/b5244e30132f4c4a30df1efe3b983873e937bdba)